projects/aca-content/src/lib/ui/variables/variables.scss

```diff
@@ -25,6 +25,7 @@ $grey-background: rgba(33, 33, 33, 0.12);
 $grey-text-background: rgba(33, 33, 33, 0.05);
 $grey-hover-background: rgba(33, 33, 33, 0.24);
 $blue-save-button-background: #1f74db;
+$blue-checkbox-background: #0a60ce;
```

Comment:
Same comment like here https://github.com/Alfresco/alfresco-content-app/pull/3908/files#r1652673368 for that file